src/main/java/org/opentripplanner/routing/api/request/ItineraryFilterParameters.java

```diff
@@ -58,12 +58,22 @@
    */
   public DoubleFunction<Double> transitGeneralizedCostLimit = null;
 
+  /**
+   * This value works the same way as {@link #transitGeneralizedCostLimit}, only for non-transit
+   * itineraries.
+    */
```

Comment:
It does not work the same way, the transit filter consider only transit this consider all itineraries. Again, document what it does - not what it does relative to something else.